requirements.txt

```diff
@@ -24,3 +24,4 @@ jinja2>=3.0.0 # BSD
 # as well as NotRequired on Python < 3.11
 typing-extensions>=4.0.1 ; python_version < '3.11' #PSF-2.0
 jsonschema>=2.6.0 # MIT
+gpg>=1.10.0
```

Comment:
The license here is missing. We haven't need to add gpg here so far or anywhere else actually.

Due to this line I get the following issue:
```
$ tox -e mypy
[...]  Installing build dependencies ... done
  Getting requirements to build wheel ... error

  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [2 lines of output]
      <string>:163: SyntaxWarning: invalid escape sequence '\s'
      Could not find gpgme-config.  Please install the libgpgme development package.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
mypy: exit 1 (6.11 seconds) /home/stefanb/keylime> python -I -m pip install mypy -r /home/stefanb/keylime/requirements.txt -r /home/stefanb/keylime/test-requirements.txt pid=48250
  mypy: FAIL code 1 (6.25 seconds)
  evaluation failed :( (6.37 seconds)
```

Reply:
`run_tests.sh` fails when `gpg`is not available. Also the github action is failing without it. It is referenced in https://github.com/keylime/keylime/blob/c26708fd3e55e2eccf98c7fe532b4a16a4426be5/keylime/signing.py#L8 How does it come that it is not needed?

Reply:
> How does it come that it is not needed?

Maybe it's being installed as a dependency for some other package but I don't know . So far we never needed it here.

Reply:
OK, let me recheck, where my issue are then coming from. Indeed the workflow does not need it, but `run_tests.sh` fails without it. I will create another version of the PR.

Reply:
I have updated the PR not manipulating the github workflows.

Reply:
I still think the gpg stuff in the installer files should be separated from this PR that tries to import a private key.

Reply:
trying to separate it

Reply:
The reason that gpg is missing from the requirements.txt is, that it needs the same gpg version installed and is normally provided by the gpg package for the distribution.
That's why there is also a comment about it in there:
https://github.com/keylime/keylime/blob/c26708fd3e55e2eccf98c7fe532b4a16a4426be5/requirements.txt#L21-L22

Reply:
I have now removed the test fixes and this PR is just for the import function.